### PR TITLE
.github: add pull request and issue template

### PR DESCRIPTION
**Pull request by xiang90**
Hopefully we do not have to ask people to change their commit message anymore...

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Bug reporting
+
+A good bug report has some very specific qualities, so please read over our short document on
+[reporting bugs](https://github.com/coreos/etcd/blob/master/Documentation/reporting_bugs.md)
```
Comment (heyitsanthony) [inline .github/ISSUE_TEMPLATE]: `../Documentation/reporting_bugs.md`?

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Contributing guidelines
+
+Please read our [contribution workflow][contributing] before submitting a pull request.
+
+[contributing]: ../Documentation/CONTRIBUTING.md#contribution-flow
```
Comment (heyitsanthony) [inline .github/PULL_REQUEST_TEMPLATE]: `../CONTRIBUTING.md#contribution-flow`?

**Comment by xiang90** [comment]
@heyitsanthony All fixed.

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**